Makefile

```diff
@@ -295,10 +297,19 @@ e2e-notation-setup:
 	${GITHUB_WORKSPACE}/bin/oras cp --from-oci-layout .staging/notation/notation.tar:v0 ${TEST_REGISTRY}/notation:unsigned
 	rm .staging/notation/notation.tar
 
+	printf 'FROM ${ALPINE_IMAGE}\nCMD ["echo", "notation tsa signed image"]' > .staging/notation/Dockerfile
+	docker buildx create --use
+	docker buildx build --output type=oci,dest=.staging/notation/notation.tar -t notation:v0 .staging/notation
+	${GITHUB_WORKSPACE}/bin/oras cp --from-oci-layout .staging/notation/notation.tar:v0 ${TEST_REGISTRY}/notation:tsa
+	rm .staging/notation/notation.tar
+	
 	rm -rf ~/.config/notation
 	.staging/notation/notation cert generate-test --default "ratify-bats-test"
+	mkdir -p .staging/tsa
+	curl -L https://github.com/notaryproject/notation/raw/a034721a7e3088250bbb04c5fccedbfca966d49e/internal/testdata/tsaRootCA.cer --output ${GITHUB_WORKSPACE}/.staging/tsa/tsaroot.cer
 
 	NOTATION_EXPERIMENTAL=1 .staging/notation/notation sign --allow-referrers-api -u ${TEST_REGISTRY_USERNAME} -p ${TEST_REGISTRY_PASSWORD} ${TEST_REGISTRY}/notation@`${GITHUB_WORKSPACE}/bin/oras manifest fetch ${TEST_REGISTRY}/notation:signed --descriptor | jq .digest | xargs`
+	NOTATION_EXPERIMENTAL=1 .staging/notation/notation sign --timestamp-url ${TIMESTAMP_URL} --timestamp-root-cert ${GITHUB_WORKSPACE}/.staging/tsa/tsaroot.cer --allow-referrers-api -u ${TEST_REGISTRY_USERNAME} -p ${TEST_REGISTRY_PASSWORD} ${TEST_REGISTRY}/notation@`${GITHUB_WORKSPACE}/bin/oras manifest fetch ${TEST_REGISTRY}/notation:tsa --descriptor | jq .digest | xargs`
```

Comment:
I see a common pattern here with "NOTATION_EXPERIMENTAL" , curious what does this flag do?

Reply:
Use flag --oci-layout to sign the image stored in OCI layout directory referenced by hello-world@sha256xxx. To access this flag --oci-layout , set the environment variable `NOTATION_EXPERIMENTAL=1`
Here is the [LINK](https://github.com/notaryproject/notation/blob/main/specs/commandline/sign.md#:~:text=Use%20flag%20%2D%2Doci%2Dlayout%20to%20sign%20the%20image%20stored%20in%20OCI%20layout%20directory%20referenced%20by%20hello%2Dworld%40sha256xxx.%20To%20access%20this%20flag%20%2D%2Doci%2Dlayout%20%2C%20set%20the%20environment%20variable%20NOTATION_EXPERIMENTAL%3D1)